clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/CramTests.scala

```diff
@@ -412,50 +424,88 @@ trait CramTests { self: BaseIntegrationSpec =>
+        rootSource,
+        sample,
+        CramExtensions.FingerprintingSummaryMetricsExtension
+      )
+    val (fingerprintDetailMetricsSource, fingerprintingDetailMetricsContents) =
+      createMockFile(
+        rootSource,
+        sample,
+        CramExtensions.FingerprintingDetailMetricsExtension
+      )
+    val (preAdapterSummaryMetricsSource, preAdapterSummaryMetricsContents) =
+      createMockFile(rootSource, sample, CramExtensions.PreAdapterSummaryMetricsExtension)
+    val (preAdapterDetailMetricsSource, preAdapterDetailMetricsContents) =
+      createMockFile(rootSource, sample, CramExtensions.PreAdapterDetailMetricsExtension)
+    val (preBqsrSelfSmSource, preBqsrSelfSmContents) =
+      createMockFile(rootSource, sample, CramExtensions.PreBqsrSelfSMExtension)
+    val (preBqsrDepthSmSource, preBqsrDepthSmContents) =
+      createMockFile(rootSource, sample, CramExtensions.PreBqsrDepthSMExtension)
 
     val endBasename = if (changeBasename) randomId else sample
 
     val rootDestination = rootSource.parent / s"moved/$randomId/"
     val cramDestination = rootDestination / s"$endBasename${CramExtensions.CramExtension}"
     val craiDestination = rootDestination / s"$endBasename${CramExtensions.CraiExtension}"
-    val alignmentMetricsDestination = rootDestination / alignmentMetricsName
-    val fingerprintMetricsDestination = rootDestination / fingerprintMetricsName
+    val analysisFilesDestination = rootDestination / s"$endBasename${CramExtensions.AnalysisFilesTxtExtension}"
+    val alignmentSummaryMetricsDestination = rootDestination / s"$endBasename${CramExtensions.AlignmentSummaryMetricsExtension}"
+    val bamValidationReportDestination = rootDestination / s"$endBasename${CramExtensions.BamValidationReportExtension}"
+    val cramValidationReportDestination = rootDestination / s"$endBasename${CramExtensions.CramValidationReportExtension}"
+    val crosscheckDestination = rootDestination / s"$endBasename${CramExtensions.CrossCheckExtension}"
+    val duplicateMetricsDestination = rootDestination / s"$endBasename${CramExtensions.DuplicateMetricsExtension}"
+    val fingerprintVcfDestination = rootDestination / s"$endBasename${CramExtensions.FingerprintVcfExtension}"
+    val fingerprintSummaryMetricsDestination = rootDestination / s"$endBasename${CramExtensions.FingerprintingSummaryMetricsExtension}"
+    val fingerprintDetailMetricsDestination = rootDestination / s"$endBasename${CramExtensions.FingerprintingDetailMetricsExtension}"
+    val preAdapterSummaryMetricsDestination = rootDestination / s"$endBasename${CramExtensions.PreAdapterSummaryMetricsExtension}"
+    val preAdapterDetailMetricsDestination = rootDestination / s"$endBasename${CramExtensions.PreAdapterDetailMetricsExtension}"
+    val preBqsrSelfSmDestination = rootDestination / s"$endBasename${CramExtensions.PreBqsrSelfSMExtension}"
+    val preBqsrDepthSmDestination = rootDestination / s"$endBasename${CramExtensions.PreBqsrDepthSMExtension}"
```

Comment:
I feel bad asking to add more, but I'd also want to check:
1. All readgroup-level metrics (stored as an array in Cram metadata) get moved
2. All logs (pointed to by a top-level directory) get moved

Should be possible to test with 4 extra files.